pydicom/filereader.py

```diff
@@ -358,14 +382,19 @@ def data_element_generator(fp,
                     # Store the encoding value in the generator for use
                     # with future elements (SQs)
```

Comment:
@darcymason  - I meant this part (lines 376-386).
But you are right, it is harmless.